### ISSUE TITLE
Scripting not working on Nexus 5

### ISSUE THREAD

**Issue by jasoncooke** [mpcjanssen/simpletask-android#237]
Firstly, LOVE the app.
I have installed it on my Nexus 10 and my Nexus 5.  The scripting is fine on the N10.  When I type the same script on the N5 and activate it, I get no results.  When I edit the script and run it against a test task, I get 

[string "Script"]:2:                      error
unexpected symbol
275 (e)

Can you advise?

Thanks

**Comment by mpcjanssen** [comment]
Are these the same app versions? Scripting has changed from javascript to lua recently. What is the exact script?

**Comment by jasoncooke** [comment]
Hi Mark
Thanks for the quick reply...

Version # is 5.3.4 (1551) on both.

Script is:

(hmmm, can't copy of on this phone) will send from tablet...

Jason

**Comment by jasoncooke** [comment]
Here's the script :

result = false
if (threshold ~= nil)  then
    result = os.time() > threshold
end
return result

Jason Cooke

**Comment by mpcjanssen** [comment]
Hi Jason,

This should work

result = false
if threshold ~= nil then
result = os.time() > threshold
end
return result

**Comment by jasoncooke** [comment]
Hi Mark,
I agree, it looks like it should, but it just does not.  I removed the
brackets and now I am getting

[string "script"]:2
syntax error

So I think that is some progress.  Strangely though, the brackets are in
place on the tablet and it works fine there.  I continue to scratch my head
and worry I am missing something simple and obvious...

and... I found it...

I had "If" instead of "if", must have been the auto-capitalization.

Sorry for taking up your time, but good to know someone is there.

Thanks again

Jason
